numpy/core/src/multiarray/compiled_base.c

```diff
@@ -140,7 +151,7 @@ arr_bincount(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
     minmax(numbers, len, &mn, &mx);
     if (mn < 0) {
         PyErr_SetString(PyExc_ValueError,
-                "The first argument of bincount must be non-negative");
+                "'list' argument must have no negative elements");
```

Comment:
Documentation calls this argument 'x', but `bincount(x=1)` complains that no `list` argument was passed.

Do we change the docs, or the real kwarg name?